core/src/test/java/jenkins/util/ResourceBundleUtilTest.java

```diff
@@ -54,6 +68,9 @@ public void test_known_locale() {
      */
     @Test
     public void test_unknown_locale() {
+        //Set Default-Locale to english
+        Locale.setDefault(new Locale("en", "US"));
+
```

Comment:
No, there is no other test concerned in this class. But if I put it into the test method, like this, the restore of the default-locale does not get executed in case of an assertion failure: 
```
 public void test_unknown_locale() {
        //Save default locale to temp
        Locale defaultOSLocale = Locale.getDefault();

        //Set Default-Locale to english
        Locale.setDefault(new Locale("en", "US"));
        JSONObject bundle = ResourceBundleUtil.getBundle("hudson.logging.Messages", new Locale("kok")); // konkani
        Assert.assertEquals("Initialing log recorders", bundle.getString("LogRecorderManager.init"));
       
        //Restore default locale from temp which does not get executed in case of 
        //assertion failure        
        Locale.setDefault(defaultOSLocale);
}
```
The other suggestion would be to put it into @Before and @After but then it gets executed 4 times instead of once.